presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -364,6 +364,7 @@ public enum RandomizeOuterJoinNullKeyStrategy
     {
         DISABLED,
         KEY_FROM_OUTER_JOIN, // Enabled only when join keys are from output of outer joins
+        COST_BASED,
```

Comment:
do we plan to make this the default value?

Reply:
Yes, will make this to be default after changes rolled out.